test/tests.py

```diff
@@ -312,8 +312,8 @@ def _():
     files = {
         "bitcoind": "var/lib/bitcoind/test/wallet.dat",
         "clightning": "var/lib/clightning/bitcoin/hsm_secret",
-        "lnd": "secrets/lnd-seed-mnemonic",
-        "joinmarket": "secrets/jm-wallet-seed",
+        "lnd": "var/lib/lnd/lnd-seed-mnemonic",
+        "joinmarket": "var/lib/joinmarket/jm-wallet-seed",
         "btcpayserver": "var/backup/postgresql/btcpaydb.sql.gz",
```

Comment:
Maybe good to also test `"secretsDir": "secrets/jm-wallet-password",`

Reply:
[Done](https://github.com/fort-nix/nix-bitcoin/pull/336/commits/b9439e4d939c3364939d202c4492a7e0584dce4d).